CoreFoundation/Base.subproj/ForFoundationOnly.h

```diff
@@ -13,6 +13,13 @@
 #if !defined(__COREFOUNDATION_FORFOUNDATIONONLY__)
 #define __COREFOUNDATION_FORFOUNDATIONONLY__ 1
 
+/// FreeBSD requires this declaration on the top. Some other include below this
```

Comment:
Can you also investigate here? In general, I don't think it's wise to mess with system imports in the CF headers, or to redefine system types, until and unless we are 100% sure that what's happening is not avoidable.